libres/lib/analysis/ies/ies_enkf_config.cpp

```diff
@@ -207,3 +210,18 @@ void ies_enkf_config_set_ies_logfile(ies_enkf_config_type *config,
 /*------------------------------------------------------------------------------------------------*/
 /* FREE_CONFIG */
 void ies_enkf_config_free(ies_enkf_config_type *config) { free(config); }
+
+double
+ies_enkf_config_calculate_steplength(const ies_enkf_config_type *ies_config,
+                                     int iteration_nr) {
+    double ies_max_step = ies_enkf_config_get_ies_max_steplength(ies_config);
+    double ies_min_step = ies_enkf_config_get_ies_min_steplength(ies_config);
+    double ies_decline_step =
+        std::max(ies_enkf_config_get_ies_dec_steplength(ies_config), 1.1);
+
+    double ies_steplength =
```

Comment:
Is this a standard way to calculate step length?

Reply:
Probably - have added a reference.